### PR TITLE
Document membership libraries

### PR DESCRIPTION
**Pull request by GeorgeTaveras1231**

### CODE REVIEW

**Comment by hsbacot** [comment]
Could we mention earlier in that docs that there are two types of configuration supported and this is an enhancement? Perhaps we include something that it isn't necessary to migrate to the webpack configuration as well.

**Comment by GeorgeTaveras1231** [comment]
@hsbacot thanks for that feedback. I went ahead and pointed out that there are two approaches higher in the documentation page (right below the **Usage**). However, I would rather not say that using the environment variables isn't required because there isn't a strategy to globally configure all instances of the client that does not use the environment variables. I would rather defer this feature until a later time. At least for now, there is a path to configure all clients.

File: docs/articles/pattern-library/utilities/membership-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    host: 'my-host',
+    apiKey: 'my-api-key'
+  }
+});
+```
+
+
+### Methods
+
+#### `#signUp` => Promise
```
Comment (marioharper) [inline docs/articles/pattern-library/utilities/membership-sdk/README.md]: Could we make this clearer that the Promise resolves with an object containing `member` and `session`? I know its documented below, but it seems like that should be documented here.

File: docs/articles/pattern-library/utilities/membership-store/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+<InstallSnippet packageJson={packageJson} />
+
+## Pre requisites
+
+Most users of this library will need configure their `CoreProvider`. Follow the instructions [here](/pattern-library/getting-started/global-dependencies#state).
+
+## Usage
+
+To configure the MembershipService, follow the instructions [here](/pattern-library/utilities/membership-store)
```
Comment (marioharper) [inline docs/articles/pattern-library/utilities/membership-store/README.md]: should this point to `/pattern-library/utilities/membership-sdk`?

**Pull request merged**